pom.xml

```diff
@@ -767,6 +767,12 @@
         <version>3.4.4</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-inline</artifactId>
```

Comment:
required to enable `Mockito.mockStatic` to replace `Powermock.mockStatic`, the latter has conflict with `Mockito.assumeTrue` as it uses a different testRunner